db/migrate/20230117150133_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :Text
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- your table should contain references this will create author_id and a corresponding index 😄 
- your table should contain references this will create author_id and a corresponding index  `(but students can also add column and index in separate commands)` 😄 👍🏾